parent/pom.xml

```diff
@@ -673,6 +673,9 @@
               <include>**/*.java</include>
               <include>**/*.scala</include>
             </includes>
+            <excludes>
+              <exclude>benchmarks/project/**/*</exclude>
```

Comment:
we had an exclude for that :eyes:

Reply:
We did? :thinking:

Reply:
I mean this:

```
 <excludes>
              <exclude>benchmarks/project/**/*</exclude>
```

Reply:
Ah you added this :man_facepalming: why we excluding that?

Reply:
Because they use different licenses, one is Apache and the other the Zeebe community license. But maybe we want to have all Zeebe community license? :man_shrugging:

Reply:
No community license should only be broker and engine as far as i know ?

Reply:
Yes, but if you remove the exclusion, since the benchmark module is not "known" by the root module anymore, it cannot override the license, so the build process will check for the default license, which is the community license =/

Reply:
ok thanks for the clarification